src/Travers.php

```diff
@@ -65,6 +76,36 @@ public function find($key)
         return $this->parseParam($this->parseKeys($key), $this->tree);
     }
 
+    /**
+     * @param       $key
+     * @param array $tree
+     *
+     * @return array|mixed
+     *
+     */
+    public function removeParam($key)
```

Comment:
Let's keep this method protected. Let's call the static method `delete` & create a non-static one `remove`. Implement it similar to `set` & `change`